cdap-ui/server/config/themes/default.json

```diff
@@ -50,6 +50,6 @@
     "namespace-security": true,
     "create-profile": true,
     "reload-system-artifacts": true,
-    "cdc": false
+    "cdc": true
```

Comment:
nit: Is this intentional? Do we want to surface CDC by default in OSS UI?

Reply:
Yes this is intentional